### ISSUE TITLE
feat: move current settings page options to profile page

### ISSUE THREAD

**Issue by matthewkurtti** [ourjapanlife/findadoc-web#1496]
# Problem
Right now, the current settings page has profile related settings (profile img, display name). Since we are creating a separate profile page, those should be moved to the profile page, and we can put some other settings on the settings page

# Success

When the above is done!

### Requirements

- [ ] Profile options are moved to the profile page
- [ ] Settings page options are replaced with some other settings options 

# Implementation notes
This will need to be paused until the following issue(s) are closed:

Issue #1495
